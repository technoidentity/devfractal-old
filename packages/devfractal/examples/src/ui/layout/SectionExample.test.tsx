import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Section } from './SectionExample'

test('Section Example', () => {
  const component: ReactTestRenderer = renderer.create(<Section />)
  expect(component.toJSON()).toMatchSnapshot()
})
