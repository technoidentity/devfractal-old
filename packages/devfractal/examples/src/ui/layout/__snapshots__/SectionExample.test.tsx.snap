// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Section with large spacing
    </h1>
    <section
      className="section is-large"
    >
      <div
        className="container"
      >
        <h1
          className="title"
        >
          Section
        </h1>
        <h1
          className="subtitle"
        >
          A simple container to divide your page into sections
        </h1>
      </div>
    </section>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Section with medium spacing
    </h1>
    <section
      className="section is-medium"
    >
      <div
        className="container"
      >
        <h1
          className="title"
        >
          Section
        </h1>
        <h1
          className="subtitle"
        >
          A simple container to divide your page into sections
        </h1>
      </div>
    </section>
  </section>,
]
`;
